Makefile

```diff
@@ -39,7 +40,8 @@ BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 BUNDLE_GEN_FLAGS ?= -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
 
 # Container runtime engine used for building the images
-BUILDER ?= podman
+BUILDER ?= docker
```

Comment:
Since we switched from docker to podman as a default container manager on all the other projects it is better to use podman by default.

If you need to use docker you can simply use the BUILDER variable without changing the Makefile

Reply:
ok